### DISCUSSION TITLE
There is no longer free plan for Meilisearch upon sing-up

### DISCUSSION THREAD

**Discussion by johnson407603** [AntlerVC/firetable#1375] [Support / Q&A] [answered]
I was trying to enable search in my Rowy project (PRO) yet realized that there are no free plans for Meilisearch anymore. Does that mean Rowy PRO account can't have search functionality activated without paying Meilisearch membership?

**Comment by hrutik7** [comment, marked as answer]
Yeah, Meilisearch recently changed their pricing plans
we're looking for some alternatives .
